presto-main/src/test/resources/catalog.json

```diff
@@ -28,9 +28,6 @@
       "catalog": "alice-catalog",
       "allow": false
     },
-    {
-      "catalog": "allowed-absent"
-    },
```

Comment:
Is there a problem with this catalog definition?
(If we do a backwards incompatible change, we need to understand the impact & document appropriately.)

Reply:
You're right.

It didn't break because jackson is "nice enough" to fill-in missing values for primitives.